admin/helpers/attendance-codes-presenter.js

```diff
@@ -10,6 +10,8 @@ const attendanceCodesPresenter = {}
  * @returns {Array} attendanceCodesPresentationData
  */
 attendanceCodesPresenter.getPresentationData = (attendanceCodes) => {
+  // The db actually has an order field, which is for the ui, but for some unknown reason,
```

Comment:
leaving a note here so we can tackle this after a discussion

Reply:
after discussion, please remove the presentation layer sorting and respect the DB order field